openapi.json

```diff
@@ -23,6 +23,17 @@
         },
         "operationId": "getRecommendationList",
         "parameters": [
+          {
+            "name": "format",
+            "in": "query",
+            "description": "Maximum number of records is 1000 i.e. 6000 rows for CSV downloads. The 'offset' parameter can be used for pagination with both formats.",
```

Comment:
To describe, request or response you can make use of Header Parameters for openapi file. 
Please refer below links:
Header parameter - https://swagger.io/docs/specification/v3_0/describing-parameters/#header-parameters
Response Media Types - https://swagger.io/docs/specification/v3_0/describing-responses/#response-media-types